tracing/providers/opentelemetry/src/main/java/io/helidon/tracing/providers/opentelemetry/package-info.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022 Oracle and/or its affiliates.
+ * Copyright (c) 2023 Oracle and/or its affiliates.
```

Comment:
This is wrong. The copyright MUST be `2022, 2023` when the file is updated and not created